apprise/plugins/NotifySlack.py

```diff
@@ -565,7 +565,7 @@ def send(self, body, title='', notify_type=NotifyType.INFO, attach=None,
                         continue
 
                 else:  # Channel
-                    result = CHANNEL_RE.match(channel)
+                    result = CHANNEL_RE.search(channel)
```

Comment:
Is there a reason you switched this back to `search()`?  `match()` is much faster and the regular expression is wrapped with `^` and `$` which invalidate the overhead `search()` causes/uses anyway.

Switch this back to `match` and we're good to go! :slightly_smiling_face: 

I have no issues at all with the regex changes.

Reply:
Sorry aout that!  Was a change I made to see if it would work with the old regex, forgot to changeback :)